recipe/run_test.sh

```diff
@@ -1,3 +1,5 @@
-cd einsteinpy_build
-if errorlevel 1 exit 1
-py.test -vv
+import matplotlib
```

Comment:
Don't you think you have written Python code in .sh file? I guess it shoudl be renamed

Reply:
I think just renaming this will pass everything!

Reply:
🙈